frontend/ui/elements/common_exit_menu_table.lua

```diff
@@ -23,6 +24,17 @@ exit_settings.restart_koreader = {
     callback = function()
         UIManager:broadcastEvent(Event:new("Restart"))
     end,
+    hold_callback = function()
+        if Device:deleteCrashLog() then
```

Comment:
Fwiw the error message will be be in the second returned result. So you might want to do something like:

```lua
local ok, error = Device:deleteCrashLog()
if ok then
```